plugin/nix-exprs/sbtix.nix

```diff
@@ -103,13 +103,30 @@ let
         );
 
 in rec {
-    mkRepo = name: artifacts: runCommand name {}
+    mkRepo = name: artifacts: localBuildsRepo: runCommand name {}
         (let
             parentDirs = filePath:
                 concatStringsSep "/" (init (splitString "/" filePath));
             linkArtifact = outputPath: urlAttrs:
                 let
-                  artifact = fetchurl urlAttrs;
+                  localpath = p:
+                    let
+                      join = l:
+                        if builtins.length(l) == 0 then ""
+                        else (
+                          if builtins.length(l) == 1 then (builtins.head l)
+                          else (builtins.head l) + "/" + (join (builtins.tail l)));
+                    in
+                      join (builtins.tail (builtins.tail (builtins.tail (builtins.tail (builtins.filter (e: builtins.typeOf e == "string") (builtins.split "/" p))))));
```

Comment:
it seems that you are trying to drop the first 3 elements of the list. can you add a comment to explain why this is needed?

Reply:
done :+1: